packages/bpk-component-image/src/withLazyLoading.js

```diff
@@ -104,11 +103,11 @@ export default function withLazyLoading(
       documentRef.removeEventListener('fullscreenchange', this.checkInView);
     };
 
-    checkInView = (): void => {
+    checkInView = throttle(() => {
```

Comment:
This is not part of fix per se, just a general good practice and because it was causing some performance issues in my tests.

Reply:
👍